### PR TITLE
Feature mes1864 increase type safety

### PR DESCRIPTION
**Pull request by rossfauldsbjss**
Part of ongoing changes to increase type safety in the Mobile App.
This one addresses Journal type safety (mes-2587)

### CODE REVIEW

**Comment by johnmorleybjss** [comment]
Assuming you haven't deleted the bundled journal type definitions because they're still in use somewhere. Just remember to remove them once we're using the centralised schema everywhere!

File: config.xml [hunk 1]
```diff
@@ -96,5 +96,5 @@
     <plugin name="cordova-plugin-network-information" spec="^2.0.1" />
     <plugin name="cordova-plugin-device-authentication" spec="https://github.com/dvsa/cordova-plugin-device-authentication" />
     <plugin name="cordova-plugin-appconfig" spec="1.1.4" />
-    <engine name="ios" spec="4.5.5" />
+    <engine name="ios" spec="^4.5.5" />
```
Comment (buzzub) [inline config.xml]: Can this be kept as 4.5.5

Comment (rossfauldsbjss) [reply config.xml]: Not sure how that got changed. But will fix it at 4.5.5

Comment (buzzub) [reply config.xml]: It changes all the time for me too.

**Pull request merged**